integration/test/basic/main_test.go

```diff
@@ -167,17 +57,25 @@ func init() {
 func TestMain(m *testing.M) {
 	ctx := context.Background()
 
+	var err error
+
 	{
-		c := e2esetup.Config{
-			HelmClient: helmClient,
-			Setup:      k8sSetup,
+		apps := []apptest.App{
+			{
+				CatalogName:   catalogName,
+				Name:          appName,
+				Namespace:     testNamespace,
+				SHA:           env.CircleSHA(),
+				ValuesYAML:    templates.CoreDNSValues,
+				WaitForDeploy: true,
+			},
 		}
-
-		v, err := e2esetup.Setup(ctx, m, c)
+		err = appTest.InstallApps(ctx, apps)
 		if err != nil {
-			l.LogCtx(ctx, "level", "error", "message", "e2e test failed", "stack", fmt.Sprintf("%#v\n", err))
+			l.LogCtx(ctx, "level", "error", "message", "install apps failed", "stack", fmt.Sprintf("%#v\n", err))
+			os.Exit(-1)
```

Comment:
Why `-1` maybe we do this in all tests and I just noticed that :D

Reply:
This is a copy and paste from other changes. Looking at https://golang.org/pkg/os/#Exit, -1 doesn't make sense. 

@rossf7 would there be a reason for using -1 here?

Reply:
The -1 exit code is so Circle CI marks the test run as failed. 

Maybe we could use t.Fatal here instead? But this is the current approach we're using.

Reply:
@rossf7  Thank you for clarifying :)